contrib/automation_tests/test_cases/symbols_tab.py

```diff
@@ -124,6 +144,25 @@ def _execute(self, function_search_string):
         wait_for_condition(lambda: '✓ F' in functions_dataview.get_item_at(0, 0).texts()[0])
 
 
+class UnhookAllFunctions(E2ETestCase):
+    """
+    Unhook all functions.
+    """
+
+    def _execute(self):
+        _show_symbols_and_functions_tabs(self.suite.top_window())
+
+        functions_dataview = DataViewPanel(self.find_control("Group", "FunctionsDataView"))
+        logging.info('Waiting for function list to be populated...')
+        wait_for_condition(lambda: functions_dataview.get_row_count() > 0, 100)
+        functions_dataview.filter.set_focus()
+        functions_dataview.filter.set_edit_text('')
+        functions_dataview.get_item_at(0, 0).click_input('left')
+        send_keys('^a^c')
```

Comment:
Can you add a comment why this is needed?

Reply:
That's "Ctrl+a" to select all function. The "^c" is a copy&paste error. Fixed and added comment.